test/sanitizer_suppressions/tsan

```diff
@@ -32,7 +32,6 @@ deadlock:CConnman::ForNode
 deadlock:CConnman::GetNodeStats
 deadlock:CChainState::ConnectTip
 deadlock:UpdateTip
-deadlock:wallet_tests::CreateWalletFromFile
```

Comment:
This could be removed after #19982 fix. But now I'd suggest s/`CreateWalletFromFile`/`CreateWallet`/.

Reply:
Ok. I've done that.